app/src/main/res/layout/fragment_invalid_password.xml

```diff
@@ -98,7 +98,7 @@
             <com.google.android.material.button.MaterialButton
                 android:id="@+id/confirmButton"
                 style="@style/TextButtonPrimary"
-                android:layout_width="match_parent"
+                android:layout_width="0dp"
```

Comment:
keep the match parent and remove the constraints

Reply:
Nope, it breaks the layout.